Backend/Meta-TV2-api/Meta-TV2-DataLayer/IDataAccess.cs

```diff
@@ -2,35 +2,47 @@
 
 public interface IDataAccess
 {
+
+
+    /// <summary>
+    /// Test method for adding to the "Groups" table.
+    /// </summary>
+    /// <param name="group">Groups object to add</param>
+    public void AddGroups(Groups group);

```

Comment:
Interface is not updated. Implementation is async

Reply:
Not necessary since, AddGroups is void